src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -664,15 +664,15 @@ internal void AddDictionaryClaims(IDictionary<string, object> claimsCollection)
                 this[kvp.Key] = kvp.Value;
         }
 
-        internal static string GetClaimValueType(object value)
+        private static string GetClaimValueType(string claimType, object value)
```

Comment:
Safe to make private?

Reply:
This one yes, it wasn't used outside of this method.